camel-openmrs/src/main/java/org/openmrs/eip/component/camel/OpenmrsAttachmentProducer.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.eip.component.camel;
+
+import java.io.File;
+
+import javax.activation.DataHandler;
+import javax.activation.FileDataSource;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.attachment.AttachmentMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+
+public class OpenmrsAttachmentProducer extends AbstractOpenmrsProducer {
+	private static Logger log = LoggerFactory.getLogger(OpenmrsAttachmentProducer.class);
```

Comment:
Looks like you did not format your code

Reply:
Oooh, I see. I'll format it this time and next